makefile

```diff
@@ -0,0 +1,59 @@
+# Download and install all dependencies packages
+#
+
+device-id-in-forwardlist := $(shell adb forward --list | wc -l)
```

Comment:
You can make a strict rule to stop forwarding whenever any devices forwarded.
But it definitely isn't a device-id-in-forwardlist, since we don't actually match device id.